### DISCUSSION TITLE
SNES inspired graphics

### DISCUSSION THREAD

**Discussion by sheepo99** [ldicker83/Micropolis-SDLPP#7] [Ideas]
Hi there, I just found about this project, and of all the Micropolis forks out there, this one seems to be the one with the most potential!

I have a suggestion relating to the graphics. Since most of the other ports tend to stick to the default graphics, I would instead like to suggest for this one to adapt something closer to the SNES Sim City graphics, which is considered to be the one of the best versions of the original Sim City. There's a very nice freely available tileset that could be used for this purpose, and since I can do a little bit of pixel art myself, I could perhaps help with the integration, and further editing.

This is the SNES inspired tileset:
https://opengameart.org/content/micropolis-snes-tiles

Let me know what you think!

**Comment by ldicker83** [comment]
I've definitely considered this. I wanted to work on the core code first and finish the port before I went making any changes but I'm not opposed to assistance with putting this together. Ultimately I'd like to make tilesets easier to work with over the way the original game did it.

**Comment by sheepo99** [comment]
Awesome! Let me know when you would like to get this started. Keep up the good work!
